io.github.ctlcltd.e2se.yaml

```diff
@@ -0,0 +1,43 @@
+app-id: io.github.ctlcltd.e2se
+runtime: org.kde.Platform
+runtime-version: '6.4'
+sdk: org.kde.Sdk
+command: e2-sat-editor
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=x11
+  - --socket=cups
+  - --filesystem=host
+rename-desktop-file: e2-sat-editor.desktop
+rename-icon: e2-sat-editor
+cleanup:
+  - '/include'
+  - '/bin/curl-config'
+  - '/lib/cmake'
+  - '/lib/debug'
```

Comment:
don't delete the debug symbols.
```suggestion
```

Reply:
From the package being built locally, '/lib/debug' is an empty directory created by compiling **libcurl4**.